.gitignore

```diff
@@ -19,3 +19,4 @@ test/version_tmp
 tmp
 *.swp
 .ruby-version
+tags
```

Comment:
No worries, I just checked it, seems that I should be using it as well. No blocking.